src/styles/globalStyles.js

```diff
@@ -33,7 +33,7 @@ export default createGlobalStyle`
         text-decoration:none;
         color:currentColor;
     }
-    h1,h2,h3,h4,h5,h6,p{
+    h1,h2,h3,h4,h5,h6,p,li{
```

Comment:
This seems strange. h tags and p, li shouldn't have the same size and weight

Reply:
should I remove this styling from global styles?

Reply:
Not the styles completely. What I mean is, that p and li are already affected by the styles on `body`. So you can remove li and p here.

The text inside the info sections should have font family Montserrat. I didn't check what you used but it seems it's Bitter here